R/apply-pgs.R

```diff
@@ -345,7 +345,12 @@ apply.polygenic.score <- function(
     sample.coordinate.to.row.dict.hash <- new.env(hash = TRUE, parent = emptyenv());
 
     for (i in 1:nrow(merged.vcf.with.pgs.data)) {
+        # skip if the variant is missing from all samples
+        if (is.na(merged.vcf.with.pgs.data[i, 'Indiv'])) {
+            next;
```

Comment:
Might be good to include a message about the skipped variant.

Reply:
This is deep in the multiallelic site handling algorithm, it's not useful to the user. Missing sites are reported in the final output.